PWGEM/PhotonMeson/TableProducer/createEMReducedEvent.cxx

```diff
@@ -98,26 +134,46 @@ struct createEMReducedEvent {
 
   void process_PCM(soa::Join<aod::Collisions, aod::EvSels> const& collisions, aod::BCs const& bcs, aod::V0Photons const& v0photons)
   {
-    const uint8_t sysflag = kPCM;
-    ;
-    process<sysflag>(collisions, bcs, v0photons, nullptr, nullptr);
+    process<kPCM>(collisions, bcs, v0photons, nullptr, nullptr);
   }
   void process_PHOS(soa::Join<aod::Collisions, aod::EvSels> const& collisions, aod::BCs const& bcs, aod::PHOSClusters const& phosclusters)
   {
-    const uint8_t sysflag = kPHOS;
-    process<sysflag>(collisions, bcs, nullptr, phosclusters, nullptr);
+    process<kPHOS>(collisions, bcs, nullptr, phosclusters, nullptr);
+  }
+  void process_EMC(soa::Join<aod::Collisions, aod::EvSels> const& collisions, aod::BCs const& bcs, aod::SkimEMCClusters const& emcclusters)
+  {
+    process<kEMC>(collisions, bcs, nullptr, nullptr, emcclusters);
   }
   void process_PCM_PHOS(soa::Join<aod::Collisions, aod::EvSels> const& collisions, aod::BCs const& bcs, aod::V0Photons const& v0photons, aod::PHOSClusters const& phosclusters)
   {
     const uint8_t sysflag = kPCM | kPHOS;
     process<sysflag>(collisions, bcs, v0photons, phosclusters, nullptr);
   }
+  void process_PCM_EMC(soa::Join<aod::Collisions, aod::EvSels> const& collisions, aod::BCs const& bcs, aod::V0Photons const& v0photons, aod::SkimEMCClusters const& emcclusters)
+  {
+    const uint8_t sysflag = kPCM | kEMC;
+    process<sysflag>(collisions, bcs, v0photons, nullptr, emcclusters);
+  }
+  void process_PHOS_EMC(soa::Join<aod::Collisions, aod::EvSels> const& collisions, aod::BCs const& bcs, aod::PHOSClusters const& phosclusters, aod::SkimEMCClusters const& emcclusters)
+  {
+    const uint8_t sysflag = kPHOS | kEMC;
+    process<sysflag>(collisions, bcs, nullptr, phosclusters, emcclusters);
+  }
+  void process_PCM_PHOS_EMC(soa::Join<aod::Collisions, aod::EvSels> const& collisions, aod::BCs const& bcs, aod::V0Photons const& v0photons, aod::PHOSClusters const& phosclusters, aod::SkimEMCClusters const& emcclusters)
+  {
+    const uint8_t sysflag = kPCM | kPHOS | kEMC;
+    process<sysflag>(collisions, bcs, v0photons, phosclusters, emcclusters);
+  }
 
   void processDummy(soa::Join<aod::Collisions, aod::EvSels> const& collisions) {}
```

Comment:
out of curiosity, whats the purpose of this?

Reply:
To have the option to run the createEMReducedEvent task with PCM, PHOS and EMCal combined. So let's say we want to cut on something for our collisions (like min Number of Photon Candidates) and we want to cross check between all methods, we would need to run this process function.